.github/workflows/gh-ci.yaml

```diff
@@ -45,7 +45,7 @@ jobs:
 
     - name: install package deps
       run: |
-        mamba install pandas pytest pytest-xdist pytest-cov codecov
+        mamba install pandas pytest pytest-xdist pytest-cov codecov pbr
```

Comment:
oops, good catch — I suppose that was previously installed by pip along the way?